test/jdk/java/nio/channels/unixdomain/Bind.java

```diff
@@ -188,6 +188,7 @@ public static void runTests() throws IOException {
             server = ServerSocketChannel.open(StandardProtocolFamily.UNIX);
             server.bind(null);
             UnixDomainSocketAddress usa = (UnixDomainSocketAddress)server.getLocalAddress();
+            usa.getPath().toFile().deleteOnExit();
```

Comment:
The test already deletes the file, I think you just want a try-finally here, same comment on a few other tests.